draft-ietf-cdni-uri-signing.xml

```diff
@@ -696,12 +750,15 @@
         <t>This section assumes the value of the CDNI Signed Token Transport (cdnistt) claim
         has been set to 1. Other values of cdnistt are out of scope of this document. </t>
 
-	<t>When using the Signed Token Renewal mechanism, the signed JWT is
+        <t>When using the Signed Token Renewal mechanism, the signed JWT is
         transported to the UA via a 'URISigningPackage' cookie added to the
         HTTP 2xx Successful message along with the content being returned to
         the UA, or to the HTTP 3xx Redirection message in case the UA is
         redirected to a different server.</t>
 
+        <t>If a Base claim is present in the JWT, the cookie should be
```

Comment:
I think either moving this to much earlier in the document, or explaining this intent much earlier would be beneficial. Now, I'm reading all the stuff above this, and have absolutely no idea why this cdnib is needed, or for.